### ISSUE TITLE
[Feature Request] Raw logging / more than two form outputs

### ISSUE THREAD

**Issue by wasp604** [bigbrodude6119/flipper-zero-evil-portal#73]
I’d like to be able to log either the raw output or multiple form outputs past just email and username, to allow a longer form with many parts.

**Comment by wasp604** [comment]
It would be helpful to be pointed toward what code deals with output parsing (that turns raw output into “u:<email> p:<password>”, to see if I could edit it to accept additional / custom form outputs.

**Comment by SneakyTaco0110** [comment]
I agree!!  I have created portals to gain more then just U: (username) & P: (Password) but the information is not output and printed and viewable from flipped

**Comment by paul-sopin** [comment]
Very late but same problem, still has not been fixed
